CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Translations](#translations)
+
+## Feature Requests
+
+Feature requests should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Ensure
+the feature request was not already reported by searching the issue tracker
+and/or mailing list. Feature requests are filed under the `enhancement` label
+on the [issue tracker](https://github.com/newsboat/newsboat/labels/enhancement).
+
+
+## Reporting Bugs
+
+**Do not open a GitHub issue if the bug is a security vulnerability**. Send an
+email to security@newsboat.org. Please encrypt your emails to 
+[PGP key 4ED6CD61932B9EBE](https://newsboat.org/newsboat.pgp)
+
+Bug reports should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Again,
+please ensure the bug was not already reported.
+
+
+## Patches / Pull Requests
+
+Patches can be submitted by sending a 
+[GitHub Pull Request](https://github.com/newsboat/newsboat/pull/new/master) or 
+emailing a patch to the Newsboat 
+[mailing list](https://groups.google.com/group/newsboat). 
+To learn more about patches, see 
+[here](https://www.kernel.org/doc/html/latest/process/submitting-patches.html) and 
+[here](https://github.com/git/git/blob/master/Documentation/SubmittingPatches). To
+learn more about pull requests, 
+see the [GitHub docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+If you are looking for a place to start contributing, take a look at the issues labeled 
+[good first issue](https://github.com/newsboat/newsboat/labels/good%20first%20issue)
+and [docs](https://github.com/newsboat/newsboat/labels/docs) on the issue tracker.
+[The Newsboat Hacker's Guide](https://github.com/newsboat/newsboat/blob/master/doc/hackers-guide.asciidoc)
```

Comment:
In the *README.md* this was a relative URL. There's probably nothing wrong with making the reference an absolute URL, I'm just curious to hear your reasoning of doing that. The link to the style guide is now absolute as well, but not the link to the docker docs down below, this is still relative. I just think that we should be consistent and ideally stick to one style.

The only advantage of a relative URL over an absolute one I can think of at the moment is, that people, reading the documents offline, **immediately** know they can have a look at other document locally, too.

Reply:
> I just think that we should be consistent and ideally stick to one style.

Agreed, I'll switch it to relative to make it more consistent considering the README also has them as relative. Absolute URLs for me are usually done by habit, since I like having the browser open the location instead of my editor.